src/systemTest/java/com/flightstats/hub/callback/CacheObject.java

```diff
@@ -0,0 +1,33 @@
+package com.flightstats.hub.callback;
+
+import com.flightstats.hub.callback.model.RequestObject;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+@Slf4j
+public class CacheObject {
+
+    private static final Map<String, List<String>> cacheRequestObject = new ConcurrentHashMap<>();
```

Comment:
Just making sure you want a global hashmap? Can an instance of this class be injected as a Singleton so you don't have a global?

Reply:
Thanks for reviewing the PR Chris. Appreciate it.

Yes I thought about singleton. It requires some setup with Guice, jersey and jetty. I am setting up jetty server to serve as a callback server. So did not want to get into the complicated stuff.
I really prefer spring framework any day which follows the mantra of convention over configuration and do not need to go thru all these hula hoops just to set up DI. But in this case I am simple did not want to introduce another framework in the current teach stack!

Reply:
Heh, different strokes I guess. I've used Spring and thought Guice was much simpler.

Reply:
Could you please share the repo url?

Reply:
https://bitbucket.flightglobal.io/projects/PDX-DS/repos/flex/browse

The different directories are different products, but there's a bunch of code shared between them. We use Dropwizard, which we have a few lines in an initalize() method to plug in Guice capability.

After that, it's just all using ```@Inject``` and, for things that aren't trivial to construct, ```@Provides``` methods. 

You can see one of our modules here. None of that may be terribly relevant though to you since the Hub has it's own framework of some sort.
https://bitbucket.flightglobal.io/projects/PDX-DS/repos/flex/browse/connections/src/main/java/com/flightstats/connections/ConnectionsMain.java
https://bitbucket.flightglobal.io/projects/PDX-DS/repos/flex/browse/connections/src/main/java/com/flightstats/connections/GuiceModule.java

Reply:
Thanks Chris. I let go of my laziness and implemented DI. Looks much better now :)
Thank you!